ui/src/utils/mod.rs

```diff
@@ -25,7 +25,9 @@ pub fn get_available_themes() -> Vec<Theme> {
         if file.metadata().unwrap().is_file() {
             let theme = file.path().display().to_string();
 
-            let theme_str = theme.split('/').last().unwrap();
+            let mut theme_str = theme.replace("\\", "/");
+            theme_str = theme_str.split('/').last().unwrap().to_string();
+
```

Comment:
Since this is a windows only issue, let's wrap this in the os tag. 

> #[cfg(target_os = "windows")]
> theme_str = theme_str...

so that we leave it relatively untouched on other systems to prevent causing bugs on other systems. (while unlikely, just nice to be sure)

Reply:
Also I'd toss in a little comment above the code to explain why we do that on windows so people like me (slightly oblivious) don't look over it in the future and break it in some other way.

Reply:
I like that! I do wish there were just "path" tools like there are for urls and other languages that abstract that stuff away. I will look around a bit. And maybe Darius's suggestion will help.

Reply:
@InfamousVague I ended up taking those two lines out and replacing them with 1 that uses those path tools that I was hopping existed. The did end up existing but instead of str you get osstr and it was a bit of madness so I changed a few other things but we are still new one less line of code so I say it's a win.